src/rgw/rgw_rest_account.cc

```diff
@@ -167,6 +167,77 @@ void RGWOp_Account_Modify::execute(optional_yield y)
                                 s->err.message, flusher, y);
 }
 
+int RGWGetAccountSummary::verify_permission(optional_yield y)
+{
+  std::string account_id;
+  if (const auto& account = s->auth.identity->get_account(); account) {
+    account_id = account->id;
+  } else {
+    return -ERR_METHOD_NOT_ALLOWED;
+  }
+
+  const RGWUserInfo& info = s->user->get_info();
```

Comment:
what about root account user, it can perform any operation without any permission explicitly attached to it